desktop/linux/release-notes/index.md

```diff
@@ -0,0 +1,29 @@
+---
+description: Change log / release notes for Docker Desktop for Linux
+keywords: Docker Desktop for Linux, release notes
+title: Docker Desktop for Linux release notes
+toc_min: 1
+toc_max: 2
+---
+
+> **Update to the Docker Desktop terms**
+>
+> Commercial use of Docker Desktop in larger enterprises (more than 250
+> employees OR more than $10 million USD in annual revenue) now requires a paid
+> subscription. The grace period for those that will require a paid subscription
+> ends on January 31, 2022. [Learn more](https://www.docker.com/blog/the-grace-period-for-the-docker-subscription-service-agreement-ends-soon-heres-what-you-need-to-know/){:
+ target="_blank" rel="noopener" class="_" id="dkr_docs_cta"}.
+{: .important}
+
```

Comment:
Given that DDL didn't exist _before_ the license changes (and this note is quite outdated as it mentions the January grace period), perhaps we should remove it here, WDYT?;

```suggestion
```